test/unit/core/client.test.ts

```diff
@@ -331,6 +331,70 @@ describe('client', function () {
     })
   })
 
+  describe('notifyAsync', function () {
+    beforeEach(() => {
+      client.configure({
+        apiKey: 'testing'
+      })
+    })
+
+    it('resolves when configured', async () => {
+      await client.notifyAsync(new Error('test'))
+    })
+
+    it('calls afterNotify in noticeable', async () => {
```

Comment:
@subzero10 just to be covered,could we add a test for the global `afterNotify()`?

Reply:
Ah yes, forgot that one. Done!